### PR TITLE
feat(dynamic-layout): add default_sort option to table, query-search-table

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [x] mirinae
  - [ ] etc 

- [ ] Apps
  - [x] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

Please review in order by commits.

- Add `default_sort` option.
- Move description of each type's spec from each of story files to one integrated story file.
  For example)
image

### CODE REVIEW

File: packages/mirinae/src/inputs/context-menu/PContextMenu.vue [hunk 1]
```diff
@@ -186,7 +186,7 @@ interface ContextMenuProps {
     searchable?: boolean;
     searchText?: string;
     readonly?: boolean;
-    resetSelectedOnUnmounted: boolean;
+    resetSelectedOnUnmounted?: boolean;
```
Comment (sulmoJ) [inline packages/mirinae/src/inputs/context-menu/PContextMenu.vue:189]: praise: 😮 😵‍💫

File: packages/mirinae/src/data-display/dynamic/dynamic-layout/templates/query-search-table/index.vue [hunk 2]
```diff
@@ -167,10 +168,10 @@ export default defineComponent<QuerySearchTableDynamicLayoutProps>({
...
 
             /** get data from fetch options */
-            sortBy: computed(() => props.fetchOptions?.sortBy || ''),
-            sortDesc: computed(() => ((props.fetchOptions?.sortDesc !== undefined) ? props.fetchOptions.sortDesc : true)),
-            pageSize: computed(() => props.fetchOptions?.pageLimit || 15),
-            queryTags: computed(() => props.fetchOptions?.queryTags || []),
+            sortBy: computed<string>(() => props.fetchOptions?.sortBy ?? props.options.default_sort?.key ?? ''),
+            sortDesc: computed<boolean>(() => props.fetchOptions?.sortDesc ?? props.options.default_sort?.desc ?? true),
+            pageSize: computed(() => props.fetchOptions?.pageLimit ?? 15),
+            queryTags: computed(() => props.fetchOptions?.queryTags ?? []),
```
Comment (yuda110) [inline packages/mirinae/src/data-display/dynamic/dynamic-layout/templates/query-search-table/index.vue:174]: priase: good!

**Review by yuda110 — APPROVED** [review]
LGTM!!

**Review by sulmoJ — APPROVED** [review]
CLEAN

**Pull request merged**